### PR TITLE
Optimize CustomModal

### PR DESCRIPTION
**Pull request by yang1666204**
## Summary

1. Optimize component CustomModal parameter passing.
2. Fix some path issues.

### CODE REVIEW

File: ui/src/pages/Login/index.tsx [hunk 1]
```diff
@@ -18,11 +18,14 @@ const Login: React.FC = () => {
...
-        navigate('/overview');
-      }
+      // Set a timer to wait for permissions to update
+      setTimeout(() => {
+        if (res.data.needReset) {
+          navigate('/reset');
+        } else {
+          navigate('/overview');
+        }
+      }, 500);
```
Comment (powerfooI) [inline ui/src/pages/Login/index.tsx:28]: Why do we need this timer to wait for permissions to be updated? What will happen if no such a timer?

Comment (yang1666204) [reply ui/src/pages/Login/index.tsx:28]: If we don't have this timer, access will not be updated after switching users with different permissions.

Comment (yang1666204) [reply ui/src/pages/Login/index.tsx:28]: Maybe I should add l `loading` to the login button.

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**